components/input-validation-mgt/org.wso2.carbon.identity.input.validation.mgt/src/main/java/org/wso2/carbon/identity/input/validation/mgt/services/InputValidationManagementService.java

```diff
@@ -74,5 +74,6 @@ List<ValidationConfiguration> getInputValidationConfiguration (String tenantDoma
      * @param tenantDomain  Tenant domain.
      * @return configuration.
      */
-    List<ValidationConfiguration> getConfigurationFromUserStore(String tenantDomain) throws InputValidationMgtException;
+    ValidationConfiguration getConfigurationFromUserStore(String tenantDomain, String field)
```

Comment:
Why do we need to change the default method here?

Reply:
We need to get default configurations from userstore only for attributes that have not configured via input validation API

Reply:
We may need to introduce a new method, but should not remove the existing methods in an interface as it can break any external implementations.